### PR TITLE
Drawer home ab

### PR DESCRIPTION
**Pull request by austinhallock**

### CODE REVIEW

File: src/components/game_player/index.coffee [hunk 1]
```diff
@@ -31,10 +32,16 @@ module.exports = class GamePlayer
...
-        else null
+      drawer: z.observe o_game.then( (game) ->
+        unless EnvironmentService.isMobile()
+          return null
+
+        User.getExperiments().then( ({nub}) ->
+          if nub is 'goHome'
+          then z 'div.nub', new Nub()
+          else new Drawer {game}
+        ).catch log.trace
```
Comment (Zolmeister) [inline src/components/game_player/index.coffee]: redundant

**Comment by Zolmeister** [comment]
done

File: src/components/game_player/index.coffee [hunk 2]
```diff
@@ -31,10 +32,16 @@ module.exports = class GamePlayer
...
-      drawer: z.observe o_game.then (game) ->
-        if EnvironmentService.isMobile()
-        then new Drawer {game}
-        else null
+      drawer: z.observe o_game.then( (game) ->
+        unless EnvironmentService.isMobile()
+          return null
+
+        User.getExperiments().then( ({nub}) ->
+          if nub is 'goHome'
```
Comment (Zolmeister) [inline src/components/game_player/index.coffee]: snake_case

**Comment by austinhallock** [comment]
done

**Pull request merged**